less/udata/resource.less

```diff
@@ -52,6 +53,19 @@
                 margin-top: 13px;
             }
         }
+
+        &.body-only {
+            margin-top: -@vspacing;
```

Comment:
If you have a better solution wihtout breaking the whole style, I take it.
I think we should cleanup these styles.